.github/workflows/main.yml

```
@@ -47,3 +47,44 @@ jobs:
         run: "vendor/bin/phpspec run"
       - name: "Integration tests"
         run: "vendor/bin/phpunit"
+
+  frontend:
+    runs-on: "ubuntu-18.04"
+    strategy:
+      matrix:
+        php-versions: [ '7.3' ]
+
+    steps:
+      - uses: "actions/checkout@v2.0.0"
+
+      - name: "Setup PHP Action"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-versions }}"
+          extensions: "intl, xdebug, imagick, apcu"
+      - name: "Install SSH key"
+        uses: "shimataro/ssh-key-action@v2"
+        with:
+          key: "${{ secrets.SSH_KEY_CI }}"
+          name: "id_rsa"
+          known_hosts: "${{ secrets.SSH_KNOWN_HOST_AKENEO_EE }}"
+
+      # Install / Prepare
+      - name: "Add EE Composer repository"
+        run: "composer -n config repositories.ee vcs ssh://git@distribution.akeneo.com:443/pim-enterprise-dev-flagbit-gmbh-co-kg.git"
+      - name: "Install PHP dependencies"
+        run: "composer install --prefer-dist --no-interaction --optimize-autoloader --no-suggest --no-progress"
+
+      - name: "yarn install"
+        uses: "borales/actions-yarn@v2.0.0"
+        with:
+          cmd: "install"
+
+      - name: "Check for obsolete ts exports"
+        run: "yarn ts-unused-exports tsconfig.json $(find src/Resources/public -name '*.ts*') --showLineNumber"
+
+      - name: "Typescript linting"
+        run: "yarn tslint src/Resources/public/**/*.ts src/Resources/public/**/*.tsx"
+
+      - name: "Typescript linting"
```

NIT: I would suggest renaming this step or add an addendum to be able to differentiate it with the step before in line 86 on first glance.

Maybe something like this? What do you think?
```suggestion
      - name: "Typescript linting (Prettier)"
```

---

True. That's Coding Style, not linting.